R/read-gs.R

```diff
@@ -17,6 +17,183 @@ read_gs <- function(path, verbose = FALSE){
     check_data_format()
 }
 
+
+
+
+
+
+
+
+
+#' This function takes in gradebook csv and lateness csv from Canvas
+#' and aggregates them into identical format as those from Gradescope. 
+#' Note that Canvas defaults assignment due time to beginning of day in lateness csv
+#' regardless of true time of lateness. 
+#'
+#'Data is then checked for correct format.
+#'
+#'@param path_to_grades The path to Canvas gradebook csv
+#'@param path_to_lateness The path to Canvas csv for assignment Lateness
+#'
+#'@return A tibble in correct format a la Gradescope
+#'
+#'
+#'@importFrom readr read_csv
+#'@importFrom  dplyr rename rename_with select left_join slice
```

Comment:
The checks are also throwing an error because you've essentially written documentation for a function that isn't exported.... either export this function (i.e. users can access it and this documentation will show up on the pkgdown site) and include it in the pkgdown.yml or remove the documentation (you can still include this info like parameters and return items as comments within the function since it is helpful info) but this is one of the reasons your checks aren't running

Reply:
Interesting. check() did not raise this issue for me.

Reply:
You can see this in the errors from the Github Action... if you wanna run those checks locally, I believe you can run `devtools::install` and then `devtools::build_site()`